solidity/test/BancorConverter.js

```diff
@@ -721,6 +712,40 @@ contract('BancorConverter', accounts => {
         await utils.catchRevert(converter.convert(tokenAddress, reserveToken.address, 30000, 1));
     });
 
+    it('verifies that convert is allowed for a whitelisted account', async () => {
+        let converter = await initConverter(accounts, true);
+        let whitelist = await Whitelist.new();
+        await whitelist.addAddress(converter.address);
```

Comment:
@yudilevi : If we have a lot of such addresses on mainnet, then perhaps we should implement a function which takes an array (instead of paying a minimum of 21000 per address).

Reply:
it has an `addAddresses` function that receives an array